src/Storefront/Framework/Cache/CacheStore.php

```diff
@@ -205,6 +206,13 @@ private function generateCacheKey(Request $request): string
 
         $hash = 'md' . hash('sha256', $uri);
 
+        $event = new HttpCacheGenerateKeyEvent($hash, $request);
+        $this->eventDispatcher->dispatch(
+            $event,
+            CacheEvents::CACHE_GENERATE_KEY_EVENT
```

Comment:
the second parameter is deprecated by Symfony. we should avoid it

Reply:
i removed the second parameter :)